example/index.html

```diff
@@ -65,7 +65,7 @@
 
 var delegate = this;
 
-var chariot = new Chariot({
+var chariotInstance = new chariot({
```

Comment:
We could. But this line wasn't working before, I think the class was being exported as `chariot`. It could break something else if they are already using `new chariot()` and we change it.